runtime-config-linux.md

```diff
@@ -146,45 +146,128 @@ The cgroups will be created if they don't exist.
 
 ```json
-    "resources": {
-        "disableOOMKiller": false,
-        "memory": {
-            "limit": 0,
-            "reservation": 0,
-            "swap": 0,
-            "kernel": 0,
-            "swappiness": -1
-        },
-        "cpu": {
-            "shares": 0,
-            "quota": 0,
-            "period": 0,
-            "realtimeRuntime": 0,
-            "realtimePeriod": 0,
-            "cpus": "",
-            "mems": ""
-        },
-        "blockIO": {
-            "blkioWeight": 0,
-            "blkioWeightDevice": "",
-            "blkioThrottleReadBpsDevice": "",
-            "blkioThrottleWriteBpsDevice": "",
-            "blkioThrottleReadIopsDevice": "",
-            "blkioThrottleWriteIopsDevice": ""
-        },
-        "hugepageLimits": null,
-        "network": {
-            "classId": "",
-            "priorities": null
-        }
-    }
+	"disableOOMKiller": false
 ```
 
-Do not specify `resources` unless limits have to be updated.
-For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
+#### Memory
+
+```json
+	"memory": {
+		"limit": 0,
+		"reservation": 0,
+		"swap": 0,
+		"kernel": 0,
+		"swappiness": -1
+	}
+```
+
+#### CPU
+
+```json
+	"cpu": {
+		"shares": 0,
+		"quota": 0,
+		"period": 0,
+		"realtimeRuntime": 0,
+		"realtimePeriod": 0,
+		"cpus": "",
+		"mems": ""
+	}
+```
+
+#### Block IO Controller
+
+`blockIO` represents the cgroup subsystem `blkio` which implements the block io controller.
+For more information, see the [kernel cgroups documentation about `blkio`](https://www.kernel.org/doc/Documentation/cgroups/blkio-controller.txt).
+
+```json
+	"blockIO": {
+		"blkioWeight": 0,
+		"blkioLeafWeight": 0,
+		"blkioWeightDevice": [
+			{
+				"major": 8,
+				"minor": 0,
+				"weight": 500,
+				"leafWeight": 300
+			},
+			{
+				"major": 8,
+				"minor": 16,
+				"weight": 500
+			}
+		],
+		"blkioThrottleReadBpsDevice": [
+			{
+				"major": 8,
+				"minor": 0,
+				"rate": 600
+			}
+		],
+		"blkioThrottleWriteBpsDevice": null,
+		"blkioThrottleReadIOPSDevice": null,
+		"blkioThrottleWriteIOPSDevice": [
+			{
+				"major": 8,
+				"minor": 16,
+				"rate": 300
+			}
+		]
+	}
+```
+
+* **blkioWeight** (uint16, optional) - specifies per cgroup weight, this is default weight of the group on all the devices until and unless overridden by per device rule, range is from 10 to 1000
+
+* **blkioLeafWeight** (uint16, optional) - equivalents of `blkioWeight` for the purpose of deciding how much weight tasks in the given cgroup has while competing with the cgroup's child cgroups, range is from 10 to 1000
+
+* **blkioWeightDevice** (array, optional) - specifies the list of devices on which bandwidth rate limit will be applied. Next parameters can be specified per device:
+	* **major** (int64, required) - major number of the device that will be limited
+	* **minor** (int64, required) - minor number of the device that will be limited
```

Comment:
Maybe worth recycling the **major, minor** combination you touch in the devices docs above?